libgstc/python/pygstc/tcp.py

```diff
@@ -28,9 +28,9 @@
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
 # OF THE POSSIBILITY OF SUCH DAMAGE.
 
-import json
-import select
+import asyncio
 import socket
+from contextlib import asynccontextmanager
```

Comment:
This feature has a minimum python version requirement of `3.7`. I can probably rework this to not require it but it makes socket cleanup a bit cleaner IMO.